src/views/HrReports/InstitutionTypeReports/InstitutionTypeReport.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { Fragment } from 'react'
+import { useState } from 'react';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useSelector } from 'react-redux'
+import { setInstitution } from 'src/redux/actions/InstitutionType.Action'
+import { axioslogin } from 'src/views/Axios/Axios';
+import CustomReport from 'src/views/Component/CustomReport'
+import 'ag-grid-community/dist/styles/ag-grid.css'
+import 'ag-grid-community/dist/styles/ag-theme-material.css'
+import { Actiontypes } from 'src/redux/constants/action.type'
+import { ToastContainer } from 'react-toastify';
+import { warningNofity } from 'src/views/CommonCode/Commonfunc';
+
+const InstitutionTypeReport = () => {
+    /** To get stored institution type values from redux */
+    const dispatch = useDispatch();
+    useEffect(() => {
+        dispatch(setInstitution());
+    }, [])
+
+    const InstitutionType = useSelector((state) => {
+        return state.getInstitutionType.InstitutionList
+    })
+
+    /** Initiliazing values */
+    const [TableData, setTableData] = useState([]);
+    const [value, setValue] = useState(0);
+
+    /** Selction checkbox for institution type  */
+    const [columnDefs] = useState([
+        {
+            headerName: 'Institution Type',
+            field: 'inst_emp_type',
+            checkboxSelection: true,
+            headerCheckboxSelectionFilteredOnly: true,
+            headerCheckboxSelection: true,
+            resizable: true,
+        },
+    ])
+
```

Comment:
u can use useMemo for passing obj 4 avoid unwanted rerender

Reply:
done